Rakefile

```diff
@@ -425,3 +425,281 @@ end
 
 spec = Gem::Specification.find_by_name 'gettext-setup'
 load "#{spec.gem_dir}/lib/tasks/gettext.rake"
+
+namespace :pwsh do
+  desc "Generate pwsh from Bolt's command line options"
+  task :generate_module do
+    @pwsh_verbs = {
+      'apply'      => 'Invoke',
+      'convert'    => 'Convert',
+      'createkeys' => 'New',
+      'init'       => 'New',
+      'install'    => 'Install',
+      'encrypt'    => 'Protect',
+      'decrypt'    => 'Unprotect',
+      'migrate'    => 'Move',
```

Comment:
Good point.

In the conventions `migrate` is mapped to `move` because migrate implies movement of something. We aren't moving, we're upgrading/updating config files so it's better to use `update` instead. There isn't an approved `upgrade` verb, but we could use that anyway. They are more like guidelines than rules anyway.

Reply:
Changed to migrate